### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by savasy**
I made small change for citation

### CODE REVIEW

File: datasets/ttc4900/README.md [hunk 1]
```diff
@@ -51,10 +51,9 @@ Savaş Yıldırım](mailto:savasy@gmail.com)
 
 ### Dataset Summary
 
-The data set is taken from kemik group
-http://www.kemik.yildiz.edu.tr/
-The data are pre-processed for the text categorization, collocations are found, character set is corrected, and so forth.
-We named TTC4900 by mimicking the name convention of TTC 3600 dataset shared by the study http://journals.sagepub.com/doi/abs/10.1177/0165551515620551
+The data set is taken from kemik group http://www.kemik.yildiz.edu.tr/
+The data are pre-processed (noun phrase chunking etc.) for the text categorization problem by the study "A Knowledge-poor Approach to Turkish Text Categorization with a Comparative Analysis, Proceedings of CICLING 2014, Springer LNCS, Nepal, 2014"  https://link.springer.com/chapter/10.1007/978-3-642-54903-8_36
+It is named TTC4900 by mimicking the name convention of TTC 3600 dataset shared by the study http://journals.sagepub.com/doi/abs/10.1177/0165551515620551
```
Comment (lhoestq) [inline datasets/ttc4900/README.md:56]:
```suggestion
The data set is taken from [kemik group](http://www.kemik.yildiz.edu.tr/)

The data are pre-processed (noun phrase chunking etc.) for the text categorization problem by the study ["A Knowledge-poor Approach to Turkish Text Categorization with a Comparative Analysis, Proceedings of CICLING 2014, Springer LNCS, Nepal, 2014"](https://link.springer.com/chapter/10.1007/978-3-642-54903-8_36)

It is named TTC4900 by mimicking the name convention of TTC 3600 dataset shared by the [study] (http://journals.sagepub.com/doi/abs/10.1177/0165551515620551)
```
You can skip lines in markdown to begin new lines
And also use hyperlinks

File: datasets/ttc4900/README.md [hunk 2]
```diff
@@ -51,10 +51,11 @@ Savaş Yıldırım](mailto:savasy@gmail.com)
...
 
-The data set is taken from kemik group
-http://www.kemik.yildiz.edu.tr/
-The data are pre-processed for the text categorization, collocations are found, character set is corrected, and so forth.
-We named TTC4900 by mimicking the name convention of TTC 3600 dataset shared by the study http://journals.sagepub.com/doi/abs/10.1177/0165551515620551
+The data set is taken from [kemik group](http://www.kemik.yildiz.edu.tr/)
+
+The data are pre-processed (noun phrase chunking etc.) for the text categorization problem by the study ["A Knowledge-poor Approach to Turkish Text Categorization with a Comparative Analysis, Proceedings of CICLING 2014, Springer LNCS, Nepal, 2014"](https://link.springer.com/chapter/10.1007/978-3-642-54903-8_36)
+
+It is named TTC4900 by mimicking the name convention of TTC 3600 dataset shared by the [study] (http://journals.sagepub.com/doi/abs/10.1177/0165551515620551)
```
Comment (lhoestq) [inline datasets/ttc4900/README.md:58]:
```suggestion
It is named TTC4900 by mimicking the name convention of TTC 3600 dataset shared by the [study](http://journals.sagepub.com/doi/abs/10.1177/0165551515620551)
```
removing space

Comment (lhoestq) [inline datasets/ttc4900/README.md]:
```suggestion
It is named TTC4900 by mimicking the name convention of TTC 3600 dataset shared by the [study](http://journals.sagepub.com/doi/abs/10.1177/0165551515620551)
```

**Review by lhoestq — APPROVED** [review]
thanks :)

File: datasets/ttc4900/README.md (hunk 1 above)
Comment (savasy) [reply datasets/ttc4900/README.md]: I think , We better remove the last line, it's confusing

The last line starts with "It is named TTC4900 by mimicking ..." bla bla

Comment (lhoestq) [reply datasets/ttc4900/README.md]: What makes you think it's confusing ?

Comment (savasy) [reply datasets/ttc4900/README.md]: I added this line 3 years ago. However,  It feels like that The study of TTC 3600 seems to have prepared the data set, doesnt' it ?    That article was referenced just for name convention.

Comment (lhoestq) [reply datasets/ttc4900/README.md]: Ok I see. I just removed this line. You can see the new dataset card here: https://github.com/huggingface/datasets/blob/master/datasets/ttc4900/README.md

**Pull request merged**